### PR TITLE
Always use whole user name as link

### PR DESCRIPTION
**Pull request by denyskon**
Starting from #25790 this shared template only linked the username of the user if both display name and username were shown. I experienced myself always trying to click on the display name - I think it is annoying for others too.

After:

grafik
grafik

### CODE REVIEW

**Review by delvh — APPROVED** [review]
I wonder if this additional space breaks any UI…

**Review by lunny — APPROVED** [review]

File: templates/shared/user/name.tmpl [hunk 1]
```diff
@@ -1,5 +1 @@
-{{if .FullName}}
-	{{.FullName}} <span>(<a class="text primary" href="{{.HomeLink}}">{{.Name}}</a>)</span>
-{{else}}
-	<a class="text primary" href="{{.HomeLink}}">{{.Name}}</a>
-{{end}}
+<a class="text muted" href="{{.HomeLink}}">{{if .FullName}}{{.FullName}} ({{.Name}}){{else}}{{.Name}}{{end}}</a>
```
Comment (wxiaoguang) [inline templates/shared/user/name.tmpl]: For consistency, the format should be `Name (FullName)`

Reference:

1. GitHub

<details>

image

</details>

2. `search_name.tmpl`

**Review by lunny — CHANGES_REQUESTED** [review]
per https://github.com/go-gitea/gitea/pull/27815/files#r1375152948

File: templates/shared/user/name.tmpl (hunk 1 above)
Comment (denyskon) [reply templates/shared/user/name.tmpl]: Done

**Review by lunny — APPROVED** [review]

**Pull request merged**